app/src/main/res/values/strings.xml

```diff
@@ -490,10 +490,10 @@
     <string name="bad_glucose_values">Bad Glucose Values</string>
     <string name="revert_to_default">Revert to Default</string>
     <string name="filtered_values">Filtered Values</string>
-    <string name="treatment_color">Treatment Color</string>
-    <string name="treatment_color_dark">Treatment Color Dark</string>
-    <string name="predictive_color">Predictive Color</string>
-    <string name="predictive_color_dark">Predictive Color Dark</string>
+    <string name="treatment_color">Insulin On Board Color</string>
+    <string name="treatment_color_dark">Insulin Activity Color</string>
+    <string name="predictive_color">Glucose Prediction Color</string>
+    <string name="predictive_color_dark">COB Prediction Color</string>
```

Comment:
These probably could be changed if that would help users. I think if I'm expanding what appears on the graph in future then I would most likely add additional colors for that purpose. @Navid200 what do you think of these wording changes?

Reply:
I find the word "Color" included at the end redundant as this is a legend and it is only a color legend and nothing else.  
So, in [documentation](https://navid200.github.io/xDrip/docs/Legend.html), I have the exact same words minus color at the end.
But, below the color legend, on the same page, I have added an image that let's the user also see the patterns for these 4 items.  If you scroll down, you will see the graph.

If you look at the insulin activity curve, you can see that it is not easy to see.  Only clarifying what the color is may not be enough to help  a user what the curve is.  Looking at the curve, it's not easy to associate it with the square of color in the legend.  That's why I thought it's best to show a typical image.

So, I guess that's a long way of saying I don't love those wording changes.

Reply:
Yes, but "color" was in the strings before. It was in the strings before it is in the strings after. So you don't object to the *change* you'd like to make _additional_ changes. You could choose to hold up this PR in order to get everything perfect, I see the benefit in that, although it seems to me that the discussions would likely go on for ever if that was the aim; or you could see if this change at least made things better and didn't make anything worse and realize that you have no objections and will likely create a further PR yourself in the future.

(Ok, this time I'm actually going to unsubscribe from this PR. It just makes me too frustrated.)

Reply:
You keep saying that and again come back.  Is someone twisting your arm?
You may be surprised to know that you are not the only one who has precious time.  Except, some people work hard without whining too much.  
We have people here like Tzachi who have worked hard consistently and have had many many PRs rejected, not just one.  And he has never whined about it.

If you don't like how we work here, get out and don't let the door hit you on the way out.

Reply:
I'm sorry about that.  I could have been less harsh there.

It's a lot of work to have to read all proposals and approve or reject.  You still have to read it to reject it.  But, the problem is that if you have to worry about what is going to be said to you if you reject and for that reason have to worry about rejecting a proposal, that would be a very unhealthy work environment.

I want the reviewers to be able to state their opinion with no fear of how their opinion is going to be criticized.
Just because someone doesn't say anything doesn't mean they don't see your complaints.
You are welcome to make proposals.  But, if you are going to be unhappy if your proposal is rejected, please don't make any proposals.

I agree 100% that sometimes it is better to take one step forward rather than taking a leap forward.  
I was going to say that instead of COB, I have Carbs ON Board.  And by removing the word color from the end, it still fits and is not a very long phrase.

Reply:
@Navid200 a reminder that incivility is not appropriate here.

In summary:

This PR started as changes suggested by @stephencmorton as to their personal opinion as to how to improve some parts of the user interface.  It wasn't to solve any particular raised issue that I know of.

I rejected the graphical changes for aesthetic reasons. @tolot27 redid the QR code graphic with red highlight which I accepted.

I've stated my reasons for not wishing to change the generic term barcode because I feel it constrains what that menu can be used for and so limits future options. I believe the term barcode to be widely understood but if not we can continue to discuss that.

The only remaining parts to this PR is the wording for the predictive simulation colors which were originally left intentionally vague and with a view to future expansion. That time has probably passed and so it is reasonable to consider improving that wording to be better understood by users and I'm simply asking for other's opinion on this to come to a decision. I asked @Navid200 about this as they are likely far closer to understanding how users react to these settings with their experience providing support to users.

This is quite an involved process to change a few words, but I'm simply responding to repeated requests to examine this PR and user interfaces are important to get right. Normally a suggestion to make a change would arise from a specific need. I might be blind to the need to change this text as I of course already know what it means which is why I would ask others for their opinion. I don't think it is a particularly high priority but I'm willing to consider it if others do think its worthwhile.

Reply:
I would use these words:

Insulin on Board
Insulin Activity
Glucose Prediction
Carbs on Board

In addition, many don't know what "Flair Colors" mean.
So, I think it's a good idea to edit those too.  But, that could be something else:
https://github.com/NightscoutFoundation/xDrip/pull/2031

Reply:
> This is quite an involved process to change a few words, but I'm simply responding to repeated requests to examine this PR and user interfaces are important to get right. Normally a suggestion to make a change would arise from a specific need. I might be blind to the need to change this text as I of course already know what it means which is why I would ask others for their opinion. I don't think it is a particularly high priority but I'm willing to consider it if others do think its worthwhile.

@jamorham a completely understandable reaction and I doubt the OP would have bothered if he'd known it would ultimately explode into such a big discussion. I don't believe it's his (or anyone's) fault that it ended up consuming so much time though.

Are there changes to the _process_ that can be implemented to reduce the overhead to the repo managers? As a new contributor, I've been instructed to keep my PRs as limited in scope/incremental as possible, but that does increase the per-change overhead somewhat. New contributors are also inherently going to start with more "trivial" changes, due to lack of familiarity with the code base.
When it comes to things like translation for example, how do we decide what's "too trivial" to change?